recipe/meta.yaml

```
@@ -43,10 +40,9 @@ requirements:
     - pykernel >=0.1.6
     - neo4j >=5.26.0
     - ipywidgets >=8.0.0
-    # conflicts with the test section when trying to use the latest palettable 3.3.0 from the main channel
-    #- palettable >=3.3.3
+    - palettable >=3.3.3
     - matplotlib-base >=3.9.4
-    - snowflake-snowpark-python >=1.26.0
+    - snowflake-snowpark-python >=1.37.0
```

https://github.com/neo4j/python-graph-visualization/blob/0.5.0/python-wrapper/pyproject.toml#L64
```suggestion
    - snowflake-snowpark-python >=1.37.0,<2
```